examples/helloworld/publisher.cpp

```diff
@@ -31,7 +31,7 @@ int main() {
         dds::domain::DomainParticipant participant(domain::default_id());
 
         /* To publish something, a topic is needed. */
-        dds::topic::Topic<HelloWorldData::Msg> topic(participant, "ddscxx_helloworld_example");
+        dds::topic::Topic<HelloWorldData::Msg> topic(participant, "HelloWorldData_Msg");
```

Comment:
I changed this one so we could test with a `C` reader/writer. I think it makes sense to change it, but maybe @eboasson has a different opinion on the matter?

Reply:
I will change this back.